GhidraBuild/BuildFiles/JsonDoclet/build.gradle

```diff
@@ -19,7 +19,7 @@ eclipse.project.name = '_JsonDoclet'
 apply plugin: 'java-library'
 
 dependencies {
-	api "com.google.code.gson:gson:2.8.6"
+	api "com.google.code.gson:gson@2.8.9"
```

Comment:
We use the `:` style everywhere.  Would you mind sticking with that convention?

Reply:
Ok